pkgs/development/python-modules/agate-sql/default.nix

```
@@ -22,7 +22,10 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ agate sqlalchemy ];
 
-  checkInputs = [ crate nose geojson ];
+  # crate is broken with SQLAlchemy > 1.3
+  # These tests rely on crate, and make the whole build fail
+  #checkInputs = [ crate nose geojson ];
+  doCheck = false;
```

checkInputs are effectful if checkPhase is disabled
```suggestion
  # crate is broken with SQLAlchemy > 1.3
  # These tests rely on crate, and make the whole build fail
  doCheck = false;
  checkInputs = [ crate nose geojson ];
 
```

---

Indeed, fixed. Thanks :)